contracts/interfaces/IEverlong.sol

```diff
@@ -1,12 +1,20 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity 0.8.20;
-
-import { IAdmin } from "./IAdmin.sol";
-import { IPositions } from "./IPositions.sol";
+pragma solidity ^0.8.20;
 
+import { IEverlongAdmin } from "./IEverlongAdmin.sol";
+import { IEverlongEvents } from "../../contracts/interfaces/IEverlongEvents.sol";
+import { IEverlongPositions } from "./IEverlongPositions.sol";
 import { IERC4626 } from "openzeppelin/interfaces/IERC4626.sol";
```

Comment:
I think we need to reevaluate this import sorting. I also think the `IEverlongEvents` import can be simplified. This is what I'd think it should be:

```suggestion
import { IERC4626 } from "openzeppelin/interfaces/IERC4626.sol";
import { IEverlongAdmin } from "./IEverlongAdmin.sol";
import { IEverlongEvents } from "./IEverlongEvents.sol";
import { IEverlongPositions } from "./IEverlongPositions.sol";
```

Reply:
Thanks!